Makefile

```diff
@@ -77,3 +77,6 @@ dockerrun: my.env
 
 dockerstop: my.env
 	${DC} stop
+
+dockerdependencycheck: my.env
+	${DC} run crontabber ./docker/run_dependency_checks.sh
```

Comment:
I've been mulling over how to do a better job of local dev environment commands. Makefiles are ok especially when rules require other rules, but they don't easily take arguments and don't yield help and are harder to document.

At some point, I think we want to have a "standard Makefile" and put everything else into some kind of command structure that runs things, is easier to document (or self-documenting, is extensible, and flexible. Just thinking out loud.